manuscript.md

```diff
@@ -42,7 +46,35 @@
 
 ## How are interaction strengths actually inferred? 
 
-## Could we use hypergraphs and multi-layer networks to predict more interactions? 
+## Could we use hypergraphs and multi-layer networks to predict more interactions?
```

Comment:
> predict more interactions?

i think the wording of the question here could be sharpened---does "more interactions" mean improving prediction accuracy of interactions?

Reply:
I flipped the question a little bit to reflect the fact that we might want to predict these types of networks as well.